flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointITCase.java

```diff
@@ -189,7 +189,6 @@ public UnalignedCheckpointITCase(String desc, UnalignedSettings settings) {
     }
 
     @Test
-    @Ignore
```

Comment:
As I understand from the offline discussion. `Parallel union` doesn't work while cogroup is _almost_ working, but not quite - it's still failing? In that case I would keep this test ignored.

Reply:
I've fiixed the remaining issues with **cogroup** in `Fix channel indices in StreamTaskNetworkInput`.

The only failing test currently is **union**. But I commented it out so `UnalignedCheckpointITCase` should run, but shouldn't fail any builds.

I can `@Ignore` it again if you prefer.

Reply:
If the remaining tests (apart of union) are stable, let's leave it as is. I thought that cogroup is also still failing sporadically.